CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 5.0.0-alpha.0 (2021-06-11)
+
+### New features
+
+- Added support for `@angular/core@^12`. [#223](https://github.com/blackbaud/skyux-core/pull/223)
+- Updated the following singleton services to use [Angular's preferred method to become tree-shakeable](https://angular.io/guide/singleton-services#using-providedin): `SkyCoreAdapterService`, `SkyAffixService`, `SkyDockService`, `SkyDynamicComponentService`, `SkyAppFormat`, `SkyLogService`, `SkyMediaQueryService`, `MutationObserverService`, `SkyNumericService`, `SkyOverlayAdapterService`, `SkyOverlayService`, `SkyAppTitleService`, `SkyUIConfigService`, `SkyViewkeeperService`, and `SkyAppWindowRef`.
+- Deprecated the `SkyCoreAdapterModule`, `SkyDynamicComponentModule`, `SkyLogModule`, and `SkyMediaQueryModule` modules. [#223](https://github.com/blackbaud/skyux-core/pull/223)
+- Deprecated the `SkyLogService`. [#223](https://github.com/blackbaud/skyux-core/pull/223)
+
+### Breaking changes
+
+- Added `@angular/router` as a peer dependency. [#223](https://github.com/blackbaud/skyux-core/pull/223)
```

Comment:
Unsure if this is a breaking change. It's more of a bug fix, really, since the library wouldn't be consumable without the router library.

Reply:
Maybe just change the heading to "Bug fixes"?